src/main/java/core/basesyntax/strategy/IncreaseAmountOperationHandlerImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitRecord;
+
+public class IncreaseAmountOperationHandlerImpl implements OperationHandler {
+    @Override
+    public int getChangedAmount(FruitRecord record) {
+        int amount = Storage.getStorage().get(record.getFruitName());
+        int newAmount = amount + record.getAmount();
```

Comment:
Why do we need to create here new variable "amount"?

Reply:
fixed